docs/source/devguides/walkthroughs/DrivenMatlabAgent-Walkthrough.rst

```diff
@@ -72,35 +75,118 @@ Installation:
 
     Follow link: http://volttron.readthedocs.io/en/develop/install.html
 
-2. Add subtree volttron-applications under volttron/applications by using
-   the following command:
+2. Clone volttron-applications in any directory outside of your volttron
+   source directory. (Best practise: clone in the parent directory of volttron
```

Comment:
Put the best practise in a bubble note section

show the output of ls in text or image